prometheus_es_exporter/__init__.py

```diff
@@ -428,6 +436,8 @@ def conv(value):
                    'in filename order. '
                    'Can be absolute, or relative to the current working directory. '
                    '(default: ./config)')
+@click.option('--threads',
+              help='Enables concurrent query execution using the number of threads specified. If the number of threads is less than 1, throws an error.')
```

Comment:
We can get click to validate the number of threads for us.

```suggestion
@click.option('--threads', type=click.IntRange(min=1), default=1,
              help='Enables concurrent query execution using the number of threads specified. '
                   '(default: 1))
```

Reply:
(my bad - missing a closing `'` there...)

Reply:
hm, can you add the missing quote? i can't seem to figure out how to apply changes atop yours in Github

Reply:
Done - tests are passing now.